sql/parser/sql.y

```diff
@@ -861,14 +861,14 @@ iso_level:
     // Mapped to the closest supported isolation level.
     $$ = SnapshotIsolation
   }
-| REPEATABLE READ
+| SNAPSHOT
   {
-    // Mapped to the closest supported isolation level.
     $$ = SnapshotIsolation
   }
-| SNAPSHOT
+| REPEATABLE READ
   {
-    $$ = SnapshotIsolation
+    // Mapped to the closest supported isolation level.
```

Comment:
The SQL standard defines these isolation levels in terms of the minimum guarantees they provide; a conforming database could define all four as equivalent to `SERIALIZABLE`, but I think the standard requires that all four names be understood. The actual implementation of isolation levels vary so much from one DB to another that I don't think users can have any particular expectation of performance differences.